terralab/commands/pipeline_runs_commands.py

```diff
@@ -117,6 +117,8 @@ def details(job_id: str):
         LOGGER.info(
             indented(f"Completed: {format_timestamp(response.job_report.completed)}")
         )
+    if response.job_report.status == SUCCEEDED_KEY:
+        LOGGER.info(indented(f"File Download Expiration Date: {format_timestamp(response.pipeline_run_report.output_expiration_date)}"))
```

Comment:
could we remove "Date" since this is a full timestamp? and I think "File Download Expiration" is plenty clear